tests/integration/cloudformation/resources/test_lambda.py

```diff
@@ -577,6 +587,14 @@ def wait_logs():
             "$..StartingPosition",
             "$..StateTransitionReason",
             "$..Topics",
+            # resource index mismatch due to SnapStart
```

Comment:
That is an example where a new feature changes the indexing of snapshots because it matches a resource transformer. It is not ideal that we need to disable these snapshot verifications for the old provider due to an unrelated change.

Reply:
Agreed. I'm generally not super happy with the state of the resource transformer. I've added this as a reference in our internal issue tracker for snapshots

Reply:
Good to track that one. I think this might become a nasty source of flaky tests with automated test re-validation.